public/content/documentation/native/controls/radio-button.md

```diff
@@ -0,0 +1,303 @@
+   - WHEN I press the "SPACEBAR" key 
+   - THEN the radio button should be activated on iOS and Android 
+   - WHEN I press the "ENTER" key 
+   - THEN the radio button should be activated on Android 
+
+2. Desktop screenreader
+
+   - GIVEN I am using a mobile screen reader 
+   - WHEN I swipe to navigate to the radio button 
+   - THEN the focus should move to the radio button 
+   - AND the radio button's name, role, and state should be expressed 
+   - WHEN I double-tap the radio button 
+   - THEN the radio button state should toggle 
+
+3. Mobile screenreader
+
+   - GIVEN the radio button is visible on the screen 
+   - WHEN a screen reader reads the radio button 
+   - THEN its name should clearly describe its purpose and match any visible label 
+   - AND its role should be identified as a button in iOS and as a radio button in Android 
+   - AND its visible label should be grouped or associated with the radio button in a single swipe 
+   - AND express its state (disabled/dimmed, iOS: checked/unchecked, selected/unselected. Android: checked/not checked) 
+
+4. Device OS Settings
+
+   - GIVEN the text label of the radio button is visible 
+   - WHEN I adjust the device text resize setting to 200% 
+   - THEN the text label should resize up to 200% without losing information 
+
+Full information: [https://www.magentaa11y.com/MagentaA11yV2#/native-criteria/controls/radio-button](https://www.magentaa11y.com/MagentaA11yV2#/native-criteria/controls/radio-button)
+
+## Videos
+
+### iOS VoiceOver
+
+<video controls>
+  <source src="media/video/native/radio-button/radioButtonIosVoiceover.mp4" type="video/mp4">
+  Your browser does not support the video tag.
+</video>
+
+### Android Talkback
+<video controls>
+  <source src="media/video/native/radio-button/radioButtonAndroidTalkback.mp4" type="video/mp4">
+  Your browser does not support the video tag.
+</video>
+
+## iOS Developer Notes
+
+   - Radio buttons allow a user to select one item from a predefined list of options
+   - A radio button should toggle between selected and unselected states. It should not automatically navigate the user to another field or screen when selected, as that may cause a change of context. Revealing new information on the same screen as a result of activating a radio button is ok, as it is usually not a change of context
+   - Name, Role, State must be announced when focus is on the control. Announcing the label in the swipe before the radio button does not meet this requirement
+
+### Name
+
+   - Programmatic name describes the purpose of the control.
+   - If a visible text label exists, the programmatic name should match the visible text label.
+      - **Note:** Setting a programmatic name while a visible text label exists may cause VoiceOver to duplicate the announcement of the name. If this happens, hide the visible text label from VoiceOver recognization.
+   - Placeholder text is NOT the programmatic name.
+
+   - **UIKit**
```

Comment:
it looks like you're formatting on the h4s for UIKit, SwiftUI, etc. (and same for Android section) didn't occur like the Button documentation.